### ISSUE TITLE
npm install error [robotjs.target.mk:126: Release/obj.target/robotjs/src/xdisplay.o] Error 1

### ISSUE THREAD

**Issue by chaseadam** [ItsRiprod/DeskThing#3]
On Fedora 40.
Likely some upstream issue, but documenting for anyone who might run into this. Seems to be related to the RobotJS dependency which was released 5 years ago...

Commented out from package.json and appears to workaround the issue.

**Comment by ItsRiprod** [comment]
Yup - RobotJS is no longer used and wasn't removed in the first two releases (v0.2.0 and v0.3.0) 

Doing `npm uninstall robotsjs` works 

Additionally, midi is not cross compatible. So for people not on windows, do `npm uninstall midi` as well and remove any reference to Launchpad in the server

Overall, using those two releases is not encouraged nor supported - I would suggest using the most recent prerelease (more apps will be supported on there VERY shortly)
